types/react-data-grid/index.d.ts

```diff
@@ -544,12 +578,46 @@ declare namespace AdazzleReactDataGrid {
 }
 
 declare namespace AdazzleReactDataGridPlugins {
-    // TODO: refine types for these addons
+    interface AutoCompleteEditorProps {
+        onCommit?: () => void;
+        options?: Array<{ id: number; title: string }>;
+        label?: any;
+        value?: any;
+        height?: number;
+        valueParams?: string[];
+        column?: AdazzleReactDataGrid.ExcelColumn;
+        resultIdentifier?: string;
+        search?: string;
+        onKeyDown?: () => void;
+        onFocus?: () => void;
+        editorDisplayValue?: (column: AdazzleReactDataGrid.ExcelColumn, value: any) => string;
+    }
+
+    interface AutoCompleteTokensEditorProps {
+        options: Array<string> | Array<{ id: number; caption: string }>;
```

Comment:
Looking at the [AutoCompleteTokensEditor code](https://github.com/adazzle/react-data-grid/blob/426801f0fd7d4236977cfdb4b89dc1ab19c07586/packages/react-data-grid-addons/src/editors/AutoCompleteTokensEditor.js#L46-L52), I think it would be more accurate to type this as `Array<string | { id: number; caption: string }>` so people could pass in arrays containing both.

Reply:
Agreed.